src/core/eme/eme_manager.ts

```diff
@@ -80,7 +83,30 @@ function clearEMESession(mediaElement : HTMLMediaElement) : Observable<never> {
 
 // TODO More events
 export type IEMEManagerEvent =
-  IEMEWarningEvent;
+  IEMEWarningEvent |
+  IEMEInitEvent;
+
+/**
+ * Get media keys infos from key system configs then attach media keys to media element.
+ * @param {HTMLMediaElement} mediaElement
+ * @param {Array.<Object>} keySystemsConfigs
+ * @param {Object} currentMediaKeysInfos
+ * @returns {Observable}
+ */
+function initMediaKeys(
```

Comment:
:+1: 
In its own file and tested?
(that should be the new dynamic, even if it's true that there is not a high chance of failing here...)